### ISSUE TITLE
After typing a long document (having more mathematical equations), NVDA crashes and becoming too slow

### ISSUE THREAD

**Issue by Bibeksharma5** [nvaccess/nvda#19962]
### Brief summary

Hello NVAccess team, this issue is making me really struggle to type the long document in Microsoft word (with UIA enabled) having many mathematical equations. I'm using my laptop for reading and writing. As the primary screen reader, I'm using Non-Visual Desktop Access. But, when I start typing, after a writing a long (around 6-7 pages), NVDA is really crashing and becoming slow. This problem is not happening when I type only the simple texts. But, if there are mathematical equations, this is really making to struggle for writing my notes. Especially, in classroom, I'm really facing hurdles. And, another issue, that time, when I press backspace or Ctrl+Backspace to delete the last character or last word, NVDA is not announcing the deleted word or character. So, my kind request, please fix this issue. I always appreciate your contribution towards visually impaired community. NVDA is very fine in other areas. But, signifcant improvements are strongly recommended to be implemented in Microsoft office. Thank you. Regards.

### Steps to reproduce

Explained above.

### Actual behavior

NVDA is becoming slow after typing a long document which I explained above.

### Expected behavior

NVDA should run fine. No crashes and slowness should be experienced. When deleting word or character, it should announce the deleted word or character even after typing long.

### NVDA logs, crash dumps and other attachments

[nvda.log](https://github.com/user-attachments/files/26784087/nvda.log)
[nvda_crash.dmp](https://github.com/user-attachments/files/26784085/nvda_crash.dmp)
[nvda_crash_stats.txt](https://github.com/user-attachments/files/26784084/nvda_crash_stats.txt)
[nvda-old.log](https://github.com/user-attachments/files/26784086/nvda-old.log)

### NVDA type

installed copy

### NVDA version

2026.1beta11 (2026.1.0.55460)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

This issue was also in previous versions.

### Windows version

Windows 11 25H2 (AMD64) build 26200.8246

### Name and version of other software in use when reproducing the issue

Microsoft Office 2021

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool

**Comment by XLTechie** [comment]
> I have not run the System Accessibility Repair Tool

Then please do so. There is no reason to report a bug until you have tried this.

It may or may not help, but it won't hurt.

**Comment by Bibeksharma5** [comment]
Hello! running com registration fixing tool did not solve the problem.

**Comment by Qchristensen** [comment]
In NVDA's "Math" settings, how do you have "Use native math support in Word and Outlook" set?

The default is unchecked.  When unchecked, MathCat is used to read math in Word.  When checked, it uses Word's native math support rather than MathCat.

**Comment by Bibeksharma5** [comment]
I know the checkbox well. This is happening in both cases (when this is
checked and unchecked).
